### ISSUE TITLE
Autocompletion replaces .or. with .OrderSpec.

### ISSUE THREAD

**Issue by hpetriffer** [X-Sharp/XSharpPublic#1280]
In the current X# Version if I type .or. inside the if statement I get always **.OrderSpec.**

```
enum BrwCurrentMode
	Cell
	Line
end enum

class Testx
	public method GridView_CustomSummaryCalculate(sender as object) as void
		local a, b as BrwCurrentMode

		if (a == BrwCurrentMode.Cell .OrderSpec.

		return
			
end class
```

**Comment by RobertvanderHulst** [comment]
This seems to depend on the project, its settings (allowdot ?) and its references. I cannot reproduce it here.
Can you create an example project?

**Comment by hpetriffer** [comment]
It seems to be fixed already in 2.17. you can close it

**Comment by RobertvanderHulst** [comment]
Ok, thanks.
